django_project/ford3/templates/campus.html

```diff
@@ -32,6 +32,7 @@ <h4 class="mt-3 pb-2">1. Details</h4>
         </div>
```

Comment:
![image](https://user-images.githubusercontent.com/34506346/59715891-1a04c700-9215-11e9-83e0-3cad8cf18e64.png)
proof of padding